JPS_Ontology/ontology/ontoassetmanagement/OntoAssetManagement.csv

```diff
@@ -30,8 +30,38 @@ UnitPrice,Class,Price,IS-A,,,,Price per individual product or service.,https://w
 DiscountedUnitPrice,Class,Price,IS-A,,,,Discounted price of one unit of a product or service.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Discounted unit price
 TotalDiscountedPretaxPrice,Class,Price,IS-A,,,,Total price of a purchase including discount and excluding tax.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Total discounted pretax price
 HomeTotalDiscountedAfterTaxPrice,Class,Price,IS-A,,,,"Total Price of a purchase including tax and discount, converted to a local currency.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Home total discounted after-tax price
-ServiceCategory,Class,https://purl.org/p2p-o/item#ItemAttribute,IS-A,,,,An attribute of an item that describes the service category it falls under.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Service category
-ServiceCode,Class,https://purl.org/p2p-o/item#ItemAttribute,IS-A,,,,An attribute of an item that describes the service code it falls under.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Service code
+MaintenanceSchedule,Class,,,,,,"A maintenance schedule tells and records when an asset is scheduled for maintenance, when was it last maintained, who performs the maintenance etc.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Maintenance schedule
+MaintenanceTask,Class,,,,,,"A small, identifiable piece of work that indicates the actions one must take to complete a maintenance work order.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Maintenance task
+Event,Class,,,,,,"A thing that happens or takes place, especially one of importance.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Event
+AssetUpdateHistory,Class,,,,,,A record of updates applied to the information of an asset.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Asset update history
+Furniture,Class,,,,,,"Objects intended to support various human activities such as seating (e.g., stools, chairs, and sofas ), eating ( tables ), storing items, working, and sleeping (e.g., beds and hammocks ). ",https://www.theworldavatar.com/kg/ontoassetmanagement/,Furniture
+Cabinet,Class,Furniture,IS-A,,,,A case or cupboard with shelves and/or drawers for storing or displaying items.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Cabinet
+Project,Class,,,,,,"Any undertaking, carried out individually or collaboratively and possibly involving research or design, that is carefully planned to achieve a particular goal.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Project
+ResearchGrant,Class,,,,,, A grant obtained for conducting scientific research generally through a competitive process.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Research grant
+Budget,Class,,,,,,The amount of money needed or available for a purpose. ,https://www.theworldavatar.com/kg/ontoassetmanagement/,Budget
```

Comment:
How are you intending to represent the actual amount of money which is in a budget?
And I believe one Grant can have multiple Accounts, correct? Otherwise, how would we represent a Grant which comes with a certain budget split, e.g. 60% of the money reserved for manpower, 20% travel and conferences, and 20% for equipment?

Reply:
Hmmm I think it will be:
ResearchGrant hasAccount Account
Account hasBudget Budget
Budget hasBudgetCategory BudgetCategory
BudgetCategory hasRemainingAmount om:AmountOfMoney

Reply:
But currently we don't have those information like the amount of money, how many accounts are there for one grant etc so I'm not sure whether we should add those properties into the ontology?

Reply:
Maybe check with Finance how the current situation about how many accounts a grant typically has, etc.
But currently we don't even have the intended `hasRemainingAmount` or did I just miss it?

Reply:
Yes. We don't have the data for hasRemaingAmount etc. I checked with Eugene, so he said that usually there is an official account for the research grant. There could also be sub-accounts, for example when a grant is given to us, a part of the grant could be sub-awarded to another entity (e.g. NUS or NTU). This means that the other entity will have an account as well.

Reply:
So a grant can have multiple accounts and the accounts doesn't necessarily have to be held by the same entity or organization. 

This account ownership can be represented with the FIBO concepts (FIBO:AccountHolder FIBO:holds FIBO:Account). As for the accounts under a grant, we could say something like: 
ResearchGrant hasAccount Account1
                        hasAccount Account2
AccountHolder1 FIBO:holds Account1
AccountHolder2 FIBO:holds Account2
AccountHolder1 FIBO:hasIdentity Organization1
AccountHolder2 FIBO:hasIdentity Organization2